### ISSUE TITLE
(t) 4.5.4-0 Samba shares fail

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2471]
Thanks to forum member Mark93 for highlighting this issue and @FroggyFlox for the initial diagnosis. We have a path failure in our samba "root preexec = " entries where we inadvertently reference our now legacy build systems' script entry points:
(See: Replace Buildout with Poetry #2432 #2434)

## before 4.5.4-0:

> /opt/rockstor/bin

## 4.5.4-0 +:
> /opt/rockstor/.venv/bin

The following is an in-place work around post encountering this issue, provided by @FroggyFlox .
```
sed -i 's:/opt/rockstor/bin:/opt/rockstor/.venv/bin:g' /etc/samba/smb.conf
systemctl restart smb nmb
```
Canonical forum reference:
https://forum.rockstor.com/t/smb-shares-inaccesible-after-v4-5-4-0-upgrade/8627

**Comment by phillxnet** [comment]
https://github.com/rockstor/rockstor-core/blob/0249f5283203461d71093447611c762901f6f378/src/rockstor/system/samba.py#L52

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2472
